web_src/less/_user.less

```diff
@@ -51,18 +51,18 @@
         img {
           width: 100%;
         }
-        @media only screen and (max-width: 768px) {
+        @media only screen and (max-width: 767px) {
           height: 250px;
           overflow: hidden;
 
           img {
             max-height: 768px;
-            max-width: 768px;
+            max-width: 767px;
```

Comment:
This is not a media query.

Reply:
Good catch. Though we probably still want to reduce to 767px, as the container width will be that size at most?

Reply:
I think setting the max width to 767px here is reasonable